<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetTeachingWorkClass</class>
 <widget class="QMainWindow" name="NetTeachingWorkClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NetTeachingWork</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton_Login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Register">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>240</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_UserName">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Password">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>网络作业提交与批改系统</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="netteachingwork.qrc"/>
 </resources>
 <connections/>
</ui>
